cmd/system-probe/api/module/loader_linux.go

```diff
@@ -28,9 +30,14 @@ func preRegister(_ *sysconfigtypes.Config, moduleFactories []Factory) error {
 	return nil
 }
 
-func postRegister(_ *sysconfigtypes.Config, moduleFactories []Factory) error {
+func postRegister(cfg *sysconfigtypes.Config, moduleFactories []Factory) error {
 	if isEBPFRequired(moduleFactories) {
 		ebpf.FlushBTF()
 	}
+	if cfg.TelemetryEnabled && ebpf.ContentionCollector != nil {
+		if err := ebpf.ContentionCollector.Initialize(ebpf.TrackAllEBPFResources); err != nil {
+			return fmt.Errorf("failed to initialize lock contention collector: %w", err)
```

Comment:
nit: maybe change to "failed to initialize **ebpf** lock contention collector..."